app/src/main/java/edu/uc/hickmadc/customnotifications/dto/Alarm.kt

```diff
@@ -1,7 +1,13 @@
 package edu.uc.hickmadc.customnotifications.dto
 
-data class Alarm (var name : String, var desc : String, var active : Boolean, var random : Boolean, var alarm_id:Int=0) {
+data class Alarm(
+    var alarmName: String,
+    var alarmDescription: String,
+    var alarmActive: Boolean,
+    var random: Boolean,
+    var alarmId: Int = 0
+) {
     override fun toString(): String {
-        return name
+        return alarmName
```

Comment:
The context of Alarm is given from the class name, so no need to repeat it in the attribute names.  That's somewhat redundant (alarm.alarmName vs. alarm.name.  Both say the same thing; one has less typing involved.)